Mathlib/LinearAlgebra/FreeModule/PID.lean

```diff
@@ -654,11 +654,3 @@ end Ideal
 end SmithNormal
 
 end PrincipalIdealDomain
-
-/-- A set of linearly independent vectors in a module `M` over a semiring `S` is also linearly
-independent over a subring `R` of `K`. -/
-theorem LinearIndependent.restrict_scalars_algebras {R S M ι : Type*} [CommSemiring R] [Semiring S]
-    [AddCommMonoid M] [Algebra R S] [Module R M] [Module S M] [IsScalarTower R S M]
-    (hinj : Function.Injective (algebraMap R S)) {v : ι → M} (li : LinearIndependent S v) :
-    LinearIndependent R v :=
```

Comment:
Please don't forget to add deprecations for deletions in the future!

Reply:
Thanks for the reminder @b-mehta I will be stricter in future.

I presume you're commenting because you were bitten in a downstream project: many apologies!